examples/describe_consumer_groups.c

```diff
@@ -212,9 +228,10 @@ static void print_group_info(const rd_kafka_ConsumerGroupDescription_t *group) {
                          rd_kafka_Node_port(coordinator));
         }
         printf(
-            "Group \"%s\", partition assignor \"%s\", "
+            "Group \"%s\", partition assignor \"%s\", type \"%s\" "
             " state %s%s, with %" PRId32 " member(s)\n",
```

Comment:
As with other field let's add the quotes
```suggestion
            " state \"%s\"%s, with %" PRId32 " member(s)\n",
```